doc.go

```diff
@@ -30,10 +30,9 @@ The code below illustrates the usage of this package:
 		}),
 	})
 	for _, url := range urls {
-		packager.Run(url)
-	}
-	if err := packager.Err(); err != nil {
-		log.Fatal(err)
+		if err := packager.Run(url); err != nil {
+			log.Print(err)
```

Comment:
`log.Println`? `printError` in cmd/main.go adds `\n` per error.

Reply:
Also the new code won't `Exit` on error unlike the previous, which called `log.Fatal` and I'm wondering if this change is also intended (this is a sample code though).

Reply:
* `log` breaks the line without `\n`, while `fmt` does not. ([sample](https://play.golang.org/p/3ltEnrhGE4Q)]
* Although the popularity makes me less confident, this codebase has adopted `log.Print` (or `log.Printf` without `\n`) so I prefer to keep it for consistency.
* I was aware that the program would no longer be terminated; I thought it was okay because this was just sample code (as you also noticed).